openpype/pipeline/colorspace.py

```diff
@@ -601,3 +606,133 @@ def _get_imageio_settings(project_settings, host_name):
     imageio_host = project_settings.get(host_name, {}).get("imageio", {})
 
     return imageio_global, imageio_host
+
+
+def get_colorspace_settings_from_publish_context(context_data):
+    """Returns solved settings for the host context.
+
+    Args:
+        context_data (publish.Context.data): publishing context data
+
+    Returns:
+        tuple | bool: config, file rules or None
+    """
+    if "imageioSettings" in context_data:
```

Comment:
Maybe do `if "imageioSettings" in context_data and context_data["imageioSettings"]` to see that the key is not empty?